packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -46,11 +50,19 @@ class ImageInfo {
   /// A string used for debugging purpopses to identify the source of this image.
   final String? debugLabel;
 
+  /// Whether or not the [image] should be disposed of when an associated
```

Comment:
In a multi frame scenario this also appears to control whether the previous frame gets disposed when the next frame is available regardless of the listeners on the ImageStream, no?